webpack.config.js

```diff
@@ -63,6 +59,10 @@ const baseConfig = {
   }
 };
 
+if (PROD) {
+  baseConfig.optimization = {minimize: true};
```

Comment:
use `--mode production`/`--mode development` in package.json instead(for build/dev/watch scripts)

Reply:
done